archive/p/php/fizz-buzz.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+for ($i = 1; $i < 101; $i++)
+{
+  $output = '';
+
+  if ($i % 3 == 0)
+  {
+    $output .= 'Fizz';
+  }
+
+  if ($i % 5 == 0)
+  {
+    $output .= 'Buzz';
+  }
+
+  if (!$output)
+  {
+    $output = $i;
+  }
+
+  echo $output . "\n";
```

Comment:
We should be using a consistent quote style.

Also, I know the interpreter doesn't complain, but it's considered good practice (I think) to have that `? >` at the end of the file

Reply:
Both of these have their reasons. I'll work backwards through your comments:

In PHP, it's considered good practice to omit the closing PHP tag if the file only contains PHP code. (Source: [PSR-2](https://www.php-fig.org/psr/psr-2/#22-files))

As for strings, we probably could make them all the same. However, the quotes in PHP actually have a semantic difference. Single quotes treat strings literally--meaning *most* escape sequences will not be interpreted. Double quotes treat escape sequences as expected. (Source: [PHP Manual](http://php.net/manual/en/language.types.string.php))

If we still want consistent quotes, we should make them all double quotes.

Reply:
> In PHP, it's considered good practice to omit the closing PHP tag if the file only contains PHP code. (Source: PSR-2)

Fair enough. Seems like I haven't written PHP in too long :slightly_smiling_face: 

About the quotes: it's ultimately up to you, I just wanted to point out the discrepancy there.

Reply:
I'll swap the quotes over to double quotes, and we can call it even. 👍